File: core/src/main/java/forpleuvoir/dhwu1a/core/message/messageitem/AtAll.java
package forpleuvoir.dhwu1a.core.message.messageitem;

import forpleuvoir.dhwu1a.core.message.messageitem.base.MessageItem;
import forpleuvoir.dhwu1a.core.message.messageitem.base.MessageItemType;

/**
 * @author forpleuvoir
 * <p>#project_name dhwu1a
 * <p>#package forpleuvoir.dhwu1a.core.message.messageitem
 * <p>#class_name AtAll
 * <p>#create_time 2021/6/30 0:25
 */
public class AtAll extends MessageItem {
    public AtAll() {
        super(MessageItemType.AtAll);
    }

    @Override
    public String toPlainText() {
        return null;
    }
}

File: core/src/main/kotlin/forpleuvoir/dhwu1a/core/message/messageitem/AtAll.kt
package forpleuvoir.dhwu1a.core.message.messageitem

import forpleuvoir.dhwu1a.core.message.messageitem.base.MessageItem
import forpleuvoir.dhwu1a.core.message.messageitem.base.MessageItemType

/**
 * @author forpleuvoir
 *
 * #project_name dhwu1a
 *
 * #package forpleuvoir.dhwu1a.core.message.messageitem
 *
 * #class_name AtAll
 *
 * #create_time 2021/6/30 0:25
 */
class AtAll : MessageItem(MessageItemType.AtAll) {
    override fun toPlainText(): String {
        return "@全体成员"
    }
}